source/changelogs/2021-06-01-June.md

```diff
@@ -0,0 +1,27 @@
+---
+title: June 2021 Changelog
+changelog: true
+description: June 2021 Pantheon changelog
+reviewed: "2021-06-01"
+---
+
+## Platform Improvements
+
+### Drupal 8.9.15
+
+[Drupal 8.9.15](https://www.drupal.org/project/drupal/releases/8.9.15) is now available on the Pantheon platform. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates) documentation. For more information, see the [Drupal 8.9.15](https://www.drupal.org/project/drupal/releases/8.9.15) release notes.
+
+### WordPress 5.7.2
+
+[WordPress 5.7.2](https://wordpress.org/news/2021/05/wordpress-5-7-2-security-release/) is now available on the Pantheon platform. This release fixes security vulnerabilities, and users are urged to upgrade their sites immediately. Detailed information on applying and debugging core updates can be found in the [Core Updates documentation](/core-updates) documentation. For more information, see the [WordPress 5.7.2](https://wordpress.org/news/2021/05/wordpress-5-7-2-security-release/) release notes.
+
+
+## Sponsored Projects
+
+### WP-CLI 2.5.0
+
+[WP-CLI 2.5.0](https://make.wordpress.org/cli/2021/05/19/wp-cli-v2-5-0-release-notes/) is now available. This release fixes security vulnerabilities, and users are urged to upgrade their sites. For more information, see the [WP-CLI 2.5.0](https://make.wordpress.org/cli/2021/05/19/wp-cli-v2-5-0-release-notes/) release notes. Detailed information on using WP-CLI on the Pantheon Platform can be found in the [WP-CLI](/wp-cli) documentation.
```

Comment:
Composer-driven WordPress users may be in control of WP-CLI versions (I know this is possible with Drush), but WP-CLI version is usually controlled by the platform, and most users won't need to make any changes, 2.5.0 is already available.